frontend/model/contracts/chatroom.js

```diff
@@ -22,6 +22,7 @@ import {
 } from './constants.js'
 import { CHATROOM_MESSAGE_ACTION } from '~/frontend/utils/events.js'
 import { logExceptNavigationDuplicated } from '~/frontend/views/utils/misc.js'
+import { GIMessage } from '~/shared/domains/chelonia/GIMessage.js'
```

Comment:
Similarly, this isn't needed here anymore (see comment below)

Reply:
Okay. Removed...